.gdbinit

```diff
@@ -262,30 +290,36 @@ define ____printzv_contents
 	if $type == 15
 		printf "_ERROR"
 	end
-	if $type == 16
+	# 	_IS_BOOL = 18
```

Comment:
The old code has _BOOL(16)/_NUMBER(17). 
However, PHP source code changed, and my PR fixes them with new number 18 and 19.  Let's keep them here just in case, until we are 99% confident to remove them.

I will remove redundant comment.